cypress/utils/apps/data_explorer/commands.js

```diff
@@ -125,33 +125,49 @@ Cypress.Commands.add(
   }
 );
 
-Cypress.Commands.add('saveQuery', (name, description) => {
+Cypress.Commands.add('saveQueryOldUI', (name, description = ' ') => {
   cy.whenTestIdNotFound('saved-query-management-popover', () => {
     cy.getElementByTestId('saved-query-management-popover-button').click();
   });
   cy.getElementByTestId('saved-query-management-save-button').click();
 
   cy.getElementByTestId('saveQueryFormTitle').type(name);
   cy.getElementByTestId('saveQueryFormDescription').type(description);
+
+  cy.getElementByTestId('savedQueryFormSaveButton').click({ force: true });
+  cy.getElementByTestId('euiToastHeader').contains('was saved').should('be.visible');
+});
+
+Cypress.Commands.add('saveQuery', (name, description = ' ') => {
+  cy.whenTestIdNotFound('saved-query-management-popover', () => {
+    cy.getElementByTestId('saved-query-management-popover-button').click();
+  });
+  cy.getElementByTestId('saved-query-management-save-button').click();
+
+  cy.getElementByTestId('saveQueryFormTitle').type(name);
+  cy.getElementByTestId('saveQueryFormDescription').type(description);
+
+  cy.getElementByTestId('savedQueryFormSaveButton').click({ force: true });
```

Comment:
same `forc`

Reply:
I think this is actually left over from some testing. Will reverify and check.

Reply:
Confirmed, removing.